partnercenter/azext_partnercenter/models/offer.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, **kwargs):
         self.id = kwargs.get('id', None)
         self.alias = kwargs.get('alias', None)
         self.type = kwargs.get('type', None)
-        self._resource = kwargs.get('resource', None)
+        self.resource = kwargs.get('resource', None)
```

Comment:
@kevinhillinger , this variable was declared private, but was getting used publicly.

But now its different from plan.py, and plan_listing.py, which still have "_resource" (but they also do not use the variable publicly)

An alternative (though not pythonic) is to add a getter method for self._resource

Reply:
for example we could add a function

```
def durable_id(self):
  return self._resource.durable_id
```